packages/bigquery/src/table.js

```diff
@@ -258,7 +258,7 @@ Table.encodeValue_ = function(value) {
  * @param {array} rows
  * @return {array} Fields using their matching names from the table's schema.
  */
-Table.mergeSchemaWithRows_ = function(schema, rows) {
+Table.mergeSchemaWithRows_ = function(BigQuery, schema, rows) {
```

Comment:
Why do we need to pass this in? Seems like something that we could easily require.